docs/source/projects.rst

```diff
@@ -361,116 +361,6 @@ this project:
 There are also additional options for disabling the creation of a Conda environment, which can be
 useful if you quickly want to test a project in your existing shell environment.
 
-.. _databricks_execution:
-
-Run an MLflow Project on Databricks
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-You can run MLflow Projects remotely on Databricks. To use this feature, you must have an enterprise
-Databricks account (Community Edition is not supported) and you must have set up the
-`Databricks CLI <https://github.com/databricks/databricks-cli>`_. Find more detailed instructions
-in the Databricks docs
-(`Azure Databricks <https://docs.databricks.com/applications/mlflow/index.html>`_,
-`Databricks on AWS <https://docs.databricks.com/applications/mlflow/index.html>`_). A brief overview
-of how to use the feature is as follows:
```

Comment:
We can still leave this part of the section in, just link to the Databricks docs instead of inlining the Databricks docs content into OSS docs

Reply:
In particular maybe keep:

```
.. _databricks_execution:	

Run an MLflow Project on Databricks	
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^	

You can run MLflow Projects remotely on Databricks. To use this feature, you must have an enterprise	
Databricks account (Community Edition is not supported) and you must have set up the	
`Databricks CLI <https://github.com/databricks/databricks-cli>`_. Find more detailed instructions	
in the Databricks docs	
(`Azure Databricks <https://docs.databricks.com/applications/mlflow/index.html>`_,	
`Databricks on AWS <https://docs.databricks.com/applications/mlflow/index.html>`_).
```

Reply:
Good call. Is there a need for the AWS/Azure text if they link to the same place?

Reply:
@smurching Done. Github diff viewer is messed up for some reason